tests/torch/test_hybrid_converter.py

```diff
@@ -75,6 +85,13 @@ def run_hybrid_llm_test(
             assert "NoParametersFound" in error.args[0]
             pytest.skip(error.args[0])
 
+        # Check we can run the simulate locally
+        if has_pbs or not glwe_backend_installed:
+            logits_simulate = hybrid_model(inputs, fhe="simulate").logits
```

Comment:
if GLWE is enabled (and this model is fully linear), it can't be simulated - we don't yet know how to simulate the GLWE stuff